index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio Project</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo">
+          <p>My Logo</p>
+        </div>
+
+        <div class="navLinks">
+          <i class="fas fa-bars"></i>
+          <ul>
+            <li><a href="">Portfolio</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+
```

Comment:
- [ ] This section is missing the background, please kindly fix that as it is a requirement for this project to stick to the design as much as possible. 
***The design***
![omarD](https://user-images.githubusercontent.com/65852621/138248690-b5be89b7-17ac-42d6-a3e7-e50b05b9998e.png)

***Your implementation***
![omarI](https://user-images.githubusercontent.com/65852621/138248746-ce0b3eb6-f54d-4d0a-aba3-fe1c5668f52b.png)